### DISCUSSION TITLE
SO 2.3 Installation Errors

### DISCUSSION THREAD

**Discussion by Xobkah** [Security-Onion-Solutions/securityonion#1667] [Unsupported Versions]
Hello, I'm new to Security Onion. I'm trying to install 2.3.2 and am getting these errors:

[ERROR   ] Command '['systemd-run', '--scope', 'systemctl', 'start', 'wazuh-agent.service']' failed with return code: 1
[ERROR   ] stderr: Running scope as unit run-10011.scope.
Job for wazuh-agent.service failed because the control process exited with error code. See "systemctl status wazuh-agent.service" and "journalctl -xe" for details.
[ERROR   ] retcode: 1
[ERROR   ] Running scope as unit run-10011.scope.
Job for wazuh-agent.service failed because the control process exited with error code. See "systemctl status wazuh-agent.service" and "journalctl -xe" for details.
              Job for wazuh-agent.service failed because the control process exited with error code. See "systemctl status wazuh-agent.service" and "journalctl -xe" for details.
Failed:     1

When I run systemctl status wazuh-agent.service
I get:

● wazuh-agent.service - Wazuh agent
   Loaded: loaded (/etc/systemd/system/wazuh-agent.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Mon 2020-10-26 21:02:30 UTC; 56min ago
  Process: 10018 ExecStart=/usr/bin/env ${DIRECTORY}/bin/ossec-control start (code=exited, status=1/FAILURE)
    Tasks: 2
   Memory: 1.2M
   CGroup: /system.slice/wazuh-agent.service
           └─10047 /var/ossec/bin/ossec-execd

Oct 26 21:02:18 SecOni systemd[1]: Starting Wazuh agent...
Oct 26 21:02:19 SecOni env[10018]: Starting Wazuh v3.13.1...
Oct 26 21:02:19 SecOni env[10018]: Started ossec-execd...
Oct 26 21:02:30 SecOni env[10018]: ossec-agentd did not start
Oct 26 21:02:30 SecOni systemd[1]: wazuh-agent.service: control process exited, code=exited status=1
Oct 26 21:02:30 SecOni systemd[1]: Failed to start Wazuh agent.
Oct 26 21:02:30 SecOni systemd[1]: Unit wazuh-agent.service entered failed state.
Oct 26 21:02:30 SecOni systemd[1]: wazuh-agent.service failed.

**Comment by weslambert** [comment]
Is everything else running?  Upon reboot do you have the same issue?
